client/data/hooks/index.js

```diff
@@ -132,10 +101,11 @@ export const useAccountInfo = () => {
 	const getAccountInfo = async () => {
 		const info = await requestAccountInfo();
 		setAccountInfo( info );
+		return info;
 	};
 
 	useEffect( () => {
 		getAccountInfo();
 	}, [] );
-	return accountInfo;
+	return { data: accountInfo, reloadAccountInfo: getAccountInfo };
```

Comment:
I suggest you just expose `accountInfo` as `accountInfo` and not `data` - much more easy to understand when looking at it from the other end and simplifies destructuring as well.